frontend/public/module/auth.js

```diff
@@ -48,18 +48,21 @@ export const authSvc = {
 
     coFetch(window.SERVER_FLAGS.logoutURL, {
       method: 'POST',
-    }).catch(e => {
+    }).then(() => authSvc._onLogout(next)).catch(e => {
       // eslint-disable-next-line no-console
       console.error('ERROR LOGGING OUT', e);
-    }).finally(() => {
-      if (window.SERVER_FLAGS.logoutRedirect && !next) {
-        window.location = window.SERVER_FLAGS.logoutRedirect;
-      } else {
-        authSvc.login();
-      }
+      authSvc._onLogout(next);
```

Comment:
It's interesting that the transpiler doesn't catch this.  
Looks like both Safari & IE fail to support.